database-1.sql

```diff
@@ -0,0 +1,17 @@
+SELECT * FROM "employees";
+
+SELECT * FROM "employees" where department_id = 50 AND salary > 4000;
+
+SELECT * FROM "employees" where first_name LIKE '%a';
+
+SELECT * FROM "employees" where commission_pct IS NOT NULL;
```

Comment:
What about department_id condition?

Reply:
Sorry, i missed that part. Now it's done